packages/table/src/tableBodyCells.tsx

```diff
@@ -24,6 +25,11 @@ export interface ITableBodyCellsProps extends IRowIndices, IColumnIndices, IProp
     cellRenderer: ICellRenderer;
 
     /**
+     * The location of the focused cell, for setting the "focused" property on cells
```

Comment:
1. Better phrasing: "The _coordinates_ of the currently focused cell" or "The currently focused cell"? No need to explain why it's needed IMO. Or, since this is just a codebase-internal component, we can explicitly say "..., for setting the 'isFocused' prop on cells."
1. Need trailing period.